x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_expression.tsx

```diff
@@ -176,10 +176,6 @@ export function XYChart({ data, args }: XYChartProps) {
             id: getSpecId(idForLegend),
             xAccessor,
             yAccessors,
-            xScaleType: (typeof rows[0][xAccessor] === 'string'
-              ? ScaleType.Ordinal
-              : ScaleType.Linear) as ScaleType.Linear | ScaleType.Ordinal | ScaleType.Time,
-            yScaleType: ScaleType.Linear as ScaleType.Linear,
```

Comment:
Were you planning on keeping this kind of change in? I did it a bit differently but with the same effect

Reply:
It wasn't on purpose. I removed it here, your PR will take care of it.